sdk/appcenter-push/build.gradle

```diff
@@ -5,7 +5,7 @@ dependencies {
     api project(':sdk:appcenter')
     def fcm = "com.google.firebase:firebase-messaging:${rootProject.ext.firebaseLibVersion}"
     def fc = "com.google.firebase:firebase-core:${rootProject.ext.firebaseLibVersion}"
-    compileOnly fcm
+    api fcm
     compileOnly fc
```

Comment:
There was an issue in a projects not pulling firebase core and just pulling firebase messaging, can you check again? We might need to expose that one too.